saleor/warehouse/availability.py

```diff
@@ -35,12 +35,13 @@ def check_stock_quantity(variant: "ProductVariant", country_code: str, quantity:
     If so - returns None. If there is less stock then required raise InsufficientStock
     exception.
     """
-    stocks = Stock.objects.get_variant_stocks_for_country(country_code, variant)
-    if not stocks:
-        raise InsufficientStock(variant)
+    if True: #variant.track_inventory:
```

Comment:
Did you mean `if variant.track_inventory`? Because `if True` does nothing and will always pass.